tests/ObjectStorageTest.php

```diff
@@ -112,6 +112,27 @@ public function testSetSecondObject(ObjectStorage $storage) : ObjectStorage
         return $storage;
     }
 
+    public function testGetObjects()
+    {
+        $storage = new ObjectStorage();
+
+        $a = new \stdClass();
+        $b = new \stdClass();
+        $c = new \stdClass();
+
+        $objects = [$a, $b, $c];
+        foreach($objects as $key => $value) {
+            $storage->set($objects[$key], $value);
```

Comment:
Here `$objects[$key] === $value`, so you can use `$value` instead.
And for the second parameter, you can use any value; it would actually be better to set it to `null` or something else to remove any confusion.